front-packages/akeneo-design-system/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -175,7 +175,7 @@ const ProgressBar = React.forwardRef<HTMLDivElement, ProgressBarProps>(
 
     return (
       <ProgressBarContainer ref={forwardedRef} {...rest}>
-        {(title || progressLabel) && (
+        {(title?.length > 0 || progressLabel?.length > 0) && (
```

Comment:
```suggestion
        {(undefined !== title || undefined !== progressLabel) && (
           <Header>
             {title && <Title title={title} id={labelId} htmlFor={progressBarId}>
             ...
             same for progressLabel
```
Let me know what you think